hw/top_earlgrey/data/ip/chip_entropy_src_testplan.hjson

```diff
@@ -179,5 +179,74 @@
       tests: []
       bazel: []
     },
+    {
+      name: chip_sw_entropy_src_fw_observe_many_contiguous
+      desc: '''Verify that the software can observe samples fast enough to extract many contiguous samples.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Configure realistic health test threshold values for FIPS/CC compliant mode.
+            - Enable ENTROPY_SRC in FIPS mode (FIXME: is it necessary to use FIPS mode?).
+            - Enable Firmware Override - Observe.
+            - Collect 1000 samples from the observe FIFO:
+              - Verify after reading each sample that the observe FIFO overflow status has not been set.
+              - FIXME: should the SW run a health check to make sure that it is fast enough?
```

Comment:
Not really sure if I understand correctly here, but the main thing we need to know is that we can extract 1000 contiguous symbols. Like you said, this can be checked by reading [FW_OV_RD_FIFO_OVERFLOW](https://opentitan.org/book/hw/ip/entropy_src/doc/registers.html#fw_ov_rd_fifo_overflow).